go.mod

```diff
@@ -4,8 +4,14 @@ go 1.14
 
 require (
 	github.com/BurntSushi/toml v0.3.1
+	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig/v3 v3.0.2
+	github.com/cyphar/filepath-securejoin v0.2.2 // indirect
+	github.com/gobwas/glob v0.2.3 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.5.0
-	sigs.k8s.io/yaml v1.1.0
+	github.com/spf13/cobra v1.0.0
+	k8s.io/apimachinery v0.18.1 // indirect
+	k8s.io/helm v2.16.6+incompatible
```

Comment:
Do we know why this is "incompatible"?  Makes me a bit nervous, but this could be my naivete with golang.

Reply:
Don't know the exact answer , it can be for many reasons semver issue or not adopting go mod etc.